deploy.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# This script is not currently meant to be executed from within the repository.
+# Copy or symlink it to some deployment directory.
+
+set -ex
+
+# Defaults for production.
+# Example for development:
+#   PORT=8100 OPENFISCA_PORT=2000 OPENFISCA_CONFIG=development PUBLIC_URL=http://next.mes-aides.sgmap.fr ./deploy.sh aah
+TARGET_BRANCH=${1:-master}  # demo
+PORT=${PORT:-8000}  # 8100
+OPENFISCA_PORT=${OPENFISCA_PORT:-12000}  # 2000
+OPENFISCA_CONFIG=${OPENFISCA_CONFIG:-mes-aides}  # development
+PUBLIC_URL=${PUBLIC_URL:-https://mes-aides.gouv.fr}  # http://next.mes-aides.sgmap.fr
```

Comment:
So if I just pass another parameter, I can have, just using this script, an instance at any url .sgmap.fr I want ? Or do I need to create the url in another way, and then pass an appropriate parameter ?